analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -240,6 +240,8 @@ def perform_analysis(args, skip_handler, context, actions, metadata_tool,
     check_env = env.extend(context.path_env_extra,
                            context.ld_lib_path_extra)
 
+    enabled_checkers = []
```

Comment:
Do we want to mix all the checkers from all of the analyzers which are enabled?

Reply:
I chose the same format as for missing checkers a few lines above.

Reply:
Maybe @gyorb is right. It would be better to print enabled checker for each analyzers separately. Similar to this:
```
Enabled checker:
 - ClangSA: checker1, checker2
 - Clang Tidy: checker3, checker4
```